conf/messages.en

```diff
@@ -32,4 +31,12 @@ customs.emailfrontend.errors.valid-confirm-email=Tell us if this is the correct
 #timeout dialog
 customs.emailfrontend.timeout.message=For your security, we will delete your answers and you will be signed out in
 customs.emailfrontend.timeout.keep-alive-button=Get another 15 minutes
-customs.emailfrontend.timeout.sign-out-button=Sign out now
\ No newline at end of file
+customs.emailfrontend.timeout.sign-out-button=Sign out now
+
+customs.emailfrontend.you-cant-use-service-NoEnrolment.hint1=You must be enrolled to CDS to use this service.
+customs.emailfrontend.you-cant-use-service-NoEnrolment.hint2=
```

Comment:
can we not avoid having this empty key with no value within messages, feels like a typo or mistakenly deleted value?